README.md

```diff
@@ -64,10 +67,10 @@ or when you want to choose which example notebook to run from the browser do:
 jupyter notebook --notebook-dir="docs/notebooks"
 ```
 
-and select an ipython notebook (file with extension `ipynb`) to run from one of the directories.
+and select an IPython notebook (file with extension `ipynb`) to run from one of the directories.
```

Comment:
```suggestion
and select a Jupyter notebook (file with extension `ipynb`) to run from one of the directories.
```
ipython has been replaces/renamed by Jupyter. This holds on more locations, so perhaps just rename it one by one

Reply:
Also changed all the instances